include/zephyr/dt-bindings/clock/stm32f0_clock.h

```diff
@@ -27,6 +27,8 @@
 /** PLL clock */
 #define STM32_SRC_PLLCLK	0x006
 
+#define STM32_SRC_LSI		0x007
```

Comment:
Please move to `/** Fixed clocks  */` with LSE/HSI.
Apply everywhere applicable.

Reply:
Done